docs/src/build_tips.md

```diff
@@ -0,0 +1,135 @@
+# Building tips
+
+BinaryBuilder provides a convenient environment to enable cross-platform building. But, many libraries have complicated build scripts that may need to be adapted to support all of the BinaryBuilder targets.
+
+## Initiating different shell commands based on target
+
+Sometimes, you need to adapt build scripts based on the target platform. This can be done within the shell script. Here is an example:
+
+```sh
+if [[ ${target} == *-apple-* ]]; then
+    export CC=clang
+    export CXX=clang++
+    export LDFLAGS="-mmacosx-version-min=10.8"
+fi
+```
+
+Here are examples of scripts with target-specific checks:
+
+* [Keno/ReadStatBuilder](https://github.com/Keno/ReadStatBuilder/blob/master/build_tarballs.jl#L35-L39) - windows check
+* [JuliaDiffEq/SundialsBuilder](https://github.com/JuliaDiffEq/SundialsBuilder/blob/cmake-mod/build_tarballs.jl#L125-L131) - 32-bit check
+* [staticfloat/IpoptBuilder](https://github.com/staticfloat/IpoptBuilder/blob/master/build_tarballs.jl#L28-L32) - apple check
+
+It is also possible to run quite different scripts for each target by running different build scripts for different sets of targets. Here is an example where windows builds are separated from other targets:
+
+* [Keno/ZlibBuilder](https://github.com/Keno/ZlibBuilder/blob/master/build_tarballs.jl)
+
+## Autoconfigure builds
+
+Autoconfigure builds are generally quite straightforward. Here is a typical approach:
+
+```sh
+./configure --prefix=$prefix --host=${target}
+make -j${nproc}
+make install
+```
+
+Here are examples of autoconfigure build scripts:
+
+* [staticfloat/OggBuilder](https://github.com/staticfloat/OggBuilder/blob/master/build_tarballs.jl)
+* [staticfloat/NettleBuilder](https://github.com/staticfloat/NettleBuilder/blob/master/build_tarballs.jl)
+
+
+## CMake builds
+
+For CMake, the wizard will suggest a template for running CMake. Typically, this will look like:
+
+```sh
+make -DCMAKE_INSTALL_PREFIX=/ -DCMAKE_TOOLCHAIN_FILE=/opt/$target/$target.toolchain
+```
```

Comment:
`-DCMAKE_INSTALL_PREFIX=$prefix` is the new hint.  Sorry that changed recently. :)